common/src/misc.rs

```diff
@@ -79,3 +82,304 @@ impl OutboundLocalTrust {
         self.outbound_trust_scores.insert(peer_id, value);
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct LocalTrustStateResponse {
+    result: Vec<(u64, u64, f32)>,
```

Comment:
These are internal indices which are not useful to clients calling the API. We need to convert these into original peer ids which are strings